src/main/java/athena/commands/ListCommand.java

```diff
@@ -36,4 +38,21 @@ public void execute(TaskList taskList, Ui ui) {
         Timetable timetable = new Timetable(taskList, importanceFilter, forecastFilter);
         ui.printTimetable(timetable);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof ListCommand)) {
+            return false;
+        }
+        ListCommand that = (ListCommand) o;
+        return taskImportance == that.taskImportance && Objects.equals(taskForecast, that.taskForecast);
```

Comment:
I think we can change `Objects.equals(taskForecast, that.taskForecast)` to `taskForecast == that.taskForecast` since we use the `enum` now instead of a `String`.